6-Typescript/server/tests/env/random.ts

```diff
@@ -0,0 +1,21 @@
+import { ITodo } from './../../interfaces/ITodo';
+import {v4 as uuidv4} from 'uuid'
+import chance from 'chance'
+
+export const createTodo = (): ITodo => {
+    return {
+    "todoText": `test${chance().integer()}`,
+    "todoId": createTodoId(),
+    "checked": false,
+    "userId": createTodoId()
+    }
+}
```

Comment:
cool idea 🕺  
1. file name is weird, maybe something `mockSomething` makes more sense 
2. a common practice is to allow overriding the the randomized values so you can have more flexibility and control. so it would be:
```
   createMockTodo = (overrides) => 
    ({ text: chance..., userId: 'blabla', ...overrides })
```
of course it'd be with types and yada yada